reference/configuration/doctrine.rst

```diff
@@ -363,7 +363,10 @@ The following block shows all possible configuration keys:
     to find your PostgreSQL version and ``mysql -V`` to get your MySQL
     version).
     
-    Always wrap the server version number with quotes to parse it as a string
+    If you are running a MariaDB database, you should prefix the ``server_version`` 
+    with ``mariadb-`` (ex: ``server_version: mariadb-10.2.12``).
```

Comment:
small nitpicking: maybe use "e.g." instead of "ex:" for consistency with other parts of the docs